### ISSUE TITLE
translation split up?

### ISSUE THREAD

**Issue by dnoesgaard** [gbif/portal16#1045]
I'm having a hard time with the translations associated with this part: https://github.com/gbif/portal16/blob/22e546ddfa325ed429d3bdedeb7cd348c87b6ac2/app/views/pages/occurrence/key/media/media.nunjucks#L34

> Not working as expected. Try the publishers version. [Take me there]()

I assume it's split up to avoid hyperlinks in the translation, but the try+take seems superfluous. Would it be ok for me to translate it along these lines:

notWorkingTryPublishers: "Not working as expected?"
takeMeThere: "Try publisher's version"

**Comment by MortenHofft** [comment]
Yeah it just makes translations and coding a lot easier if that way. It annoys me, but I'm trying to be pragmatic.
